File: scratch/find_logo_refined.js

const fs = require('fs');
const path = require('path');

const filePath = 'c:\\Users\\sk barath\\OneDrive\\Desktop\\zyvarsportfolio\\assets\\index.js';
const content = fs.readFileSync(filePath, 'utf8');

let searchStr = 'logo.png';
let idx = content.indexOf(searchStr);
while (idx !== -1) {
    let start = Math.max(0, idx - 100);
    let end = Math.min(content.length, idx + 100);
    console.log(`Match at ${idx}: ...${content.substring(start, end)}...`);
    idx = content.indexOf(searchStr, idx + 1);
}

File: scratch/find_logo_refined.py

import os

file_path = r'c:\Users\sk barath\OneDrive\Desktop\zyvarsportfolio\assets\index.js'
with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

indices = [i for i in range(len(content)) if content.startswith('logo.png', i)]
for idx in indices:
    start = max(0, idx - 100)
    end = min(len(content), idx + 100)
    print(f"Match at {idx}: ...{content[start:end]}...")
